test/python/test.py

```diff
@@ -1,7 +1,5 @@
 from mqt import syrec
 
 if __name__ == "__main__":
-    dummy = syrec.Dummy()
-    dummy.setVal(42)
-    print(dummy.getVal())
-    dummy = syrec.Dummy(42)
+    circ = syrec.circuit()
```

Comment:
How do I test the Python part of the code (similar to how it is happening for c++ using gtest)?

Reply:
In our other projects, we use `pytest` for these purposes, see ,e.g., https://github.com/cda-tum/qcec/tree/develop/test/python. This could also be adopted here.
This requires the following steps:
- adding a `requirements-test.txt` file to the `test/python` directory that specifies the `pytest` dependency as in https://github.com/cda-tum/qcec/blob/72d3129d6e520c7cc4508d4d30b6b6fe561cac8b/test/python/requirements-test.txt#L1
- writing similar tests to the C++ ones in Python (would make sense to stick to the same structure)
- adding the `requirements-test.txt` file to the dependabot configuration (`.github/dependabot.yml`) as in https://github.com/cda-tum/qcec/blob/72d3129d6e520c7cc4508d4d30b6b6fe561cac8b/.github/dependabot.yml#L30-L37
- adding the python tests to the deployment configuration (`.github/workflows/deploy.yml`) as in https://github.com/cda-tum/qcec/blob/72d3129d6e520c7cc4508d4d30b6b6fe561cac8b/.github/workflows/deploy.yml#L71-L87